navbar with icons

stshayna opened the pull request:
1. Created basic navbar with 4 icons and a dropdown menu for login/log out.
Details, e.g. icons/links, can change after any further discussion!

2. Updated colors of messages in chatroom to align with Figma.

![Screenshot 2022-05-26 at 11 34 32 PM](https://user-images.githubusercontent.com/75033073/170624243-04310463-defc-47da-8a1b-283d64239288.png)
![Screenshot 2022-05-26 at 11 34 48 PM](https://user-images.githubusercontent.com/75033073/170624247-972595ef-5b38-41be-b383-57d389c0922c.png)
![Screenshot 2022-05-26 at 11 35 16 PM](https://user-images.githubusercontent.com/75033073/170624253-332646e8-750d-44d3-9b54-363ce96ab977.png)
![Screenshot 2022-05-26 at 11 35 28 PM](https://user-images.githubusercontent.com/75033073/170624271-bf46eef6-e3c6-4ab1-93ab-4978ae6eef38.png)

MarlAndre commented:
1. We changed the first icon for the map marker instead of the home, so it will be clearer for the user what is the purpose of that icon
2.  I think having the avatar image in the navbar is a bit weird, I would change it for an icon so we have the same continuity in the navbar

MarlAndre commented on app/views/shared/_navbar.html.erb:
```diff
@@ -1,39 +1,32 @@
-    <%= link_to "#", class: "navbar-brand" do %>
-      <%= image_tag "https://raw.githubusercontent.com/lewagon/fullstack-images/master/uikit/logo.png" %>
-    <% end %>
+    <a href="/"><i class="fas fa-home"></i></a>
```
Change home icon for map marker icon

stshayna replied:
changed the icon!

MarlAndre commented on app/views/shared/_navbar.html.erb:
```diff
@@ -1,39 +1,32 @@
     <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
-      <span class="navbar-toggler-icon"></span>
+      <% if current_user == User.first %>
+        <%= image_tag "demo_user.png", class: "avatar dropdown-toggle text-white", id: "navbarDropdown", data: { bs_toggle: "dropdown" }, 'aria-haspopup': true, 'aria-expanded': false %>
```
Having an image next to a bunch of icons is weird, I would put the profile icon back

stshayna replied:
ok, removed the profile picture as requested

MarlAndre commented on app/views/shared/_navbar.html.erb:
```diff
@@ -1,39 +1,32 @@
-              <%= link_to "Another action", "#", class: "dropdown-item" %>
-              <%= link_to "Log out", destroy_user_session_path, method: :delete, class: "dropdown-item" %>
-            </div>
+            <%= link_to "Log out", destroy_user_session_path, method: :delete %>
```
If we want to login / logout, it would make more sense to do it by clicking on the profile icon

stshayna replied:
ok, removed the profile picture as requested

MarlAndre replied:
So, if we click on the profile icon, there's a dropdown to logout ? But what if the user want to go to heir profile page, is there a route for that or we will create it later ?

stshayna replied:
There can be two ways, we can:
1. Add "Profile Page" above "Log out" in the dropdown menu; or
2. Remove the dropdown menu and replace the url by user profile page

I'm fine with both options!

MarlAndre replied:
Ok, let's talk about it tomorrow. I'll merge the branch :)

stshayna commented:
Navbar after changes:

![Screenshot 2022-05-27 at 11 51 56 AM](https://user-images.githubusercontent.com/75033073/170734784-54a45d14-847b-4af0-ae14-2d0255145cf5.png)
![Screenshot 2022-05-27 at 11 52 08 AM](https://user-images.githubusercontent.com/75033073/170734796-4e5975d7-25eb-4b14-8ca5-47c4c05c965c.png)

stshayna commented:
Navbar after changes:
1. removed profile picture
2. changed home icon to map marker icon
3. moved the 'Login"/"Log out" text to the end
![Screenshot 2022-05-27 at 11 51 56 AM](https://user-images.githubusercontent.com/75033073/170735669-6cbd6a9d-296b-4e5c-bda2-0779a7e54700.png)
![Screenshot 2022-05-27 at 11 56 06 AM](https://user-images.githubusercontent.com/75033073/170735681-923af7c6-97da-4b4b-bc24-4a7e8c98a04d.png)
